### DISCUSSION TITLE
UPDATE - 4.30 pm CET - Reminder: Europe enters the winter time

### DISCUSSION THREAD

**Discussion by xzhh** [espressopp/espressopp#468] [Team Posts]
Dear all

The starting date of winter time may differ in different countries. If there's no other suggestions, lets stick to 4.00 pm (CET) for the weekly meeting and pay attentions to your local time.

Zhenhao

**Comment by AndreBrinkmann** [comment]
Dear Zhenhao, 

  dear All,

 

this term, I will give lectures from 2:00 to 4:00 PM every Thursday and in the first weeks, I will afterwards meet with my tutors. Therefore, I cannot attend before 4:30 PM.

 

Best Regards, 

 

André

 

Von: Zhen-Hao Xu ***@***.***>
Antworten an: espressopp/discussion ***@***.***>
Datum: Mittwoch, 2. November 2022 um 10:47
An: espressopp/discussion ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: [espressopp/discussion] Reminder: Europe enters the winter time (#14)

 

Dear all

The starting date of winter time may differ in different countries. If there's no other suggestions, lets stick to 4.00 pm (CET) for the weekly meeting and pay attentions to your local time.

Zhenhao

**Comment by XzzX** [comment]
Dear All,

I am on vacation and will not attend the meeting tomorrow.
As far as my work is concerned, there are no news from Luis, so also no 
progress on my side.

Best
Sebastian

On 02.11.22 15:00, AndreBrinkmann wrote:
> Dear Zhenhao,
> 
>    dear All,
> 
> 
> 
> this term, I will give lectures from 2:00 to 4:00 PM every Thursday and 
> in the first weeks, I will afterwards meet with my tutors. Therefore, I 
> cannot attend before 4:30 PM.
> 
> 
> 
> Best Regards,
> 
> 
> 
> André
> 
> 
> 
> Von: Zhen-Hao Xu ***@***.***>
> Antworten an: espressopp/discussion ***@***.***>
> Datum: Mittwoch, 2. November 2022 um 10:47
> An: espressopp/discussion ***@***.***>
> Cc: Subscribed ***@***.***>
> Betreff: [espressopp/discussion] Reminder: Europe enters the winter time 
> (#14)
> 
> 
> 
> Dear all
> 
> The starting date of winter time may differ in different countries. If 
> there's no other suggestions, lets stick to 4.00 pm (CET) for the weekly 
> meeting and pay attentions to your local time.
> 
> Zhenhao
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message 
> ID: ***@***.***>
> 
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/orgs/espressopp/teams/discussion/discussions/14/comments/1>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAU3OTAZ3RRSK7YRK65IGF3WGJXY5ANCNFSM6AAAAAARU4KIGQ>.
> You are receiving this because you are subscribed to this thread.Message 
> ID: ***@***.***>
> 

-- 
Dr. Sebastian Eibl
Max Planck Computing and Data Facility (MPCDF)
Gießenbachstr. 2
D-85748 Garching
Germany

**Comment by stuehn** [comment]
Dear all,

due to the a CECAM (online-) talk that I would like to attend, I will 
only be available after 16:30 and though would like to shift the meeting 
until that time.

Would that be possible ? (if not, I would simply join the meeting at 16:30)

Best regards,

Torsten


On 11/2/22 10:47, Zhen-Hao Xu wrote:
> Dear all
> 
> The starting date of winter time may differ in different countries. If 
> there's no other suggestions, lets stick to 4.00 pm (CET) for the weekly 
> meeting and pay attentions to your local time.
> 
> Zhenhao
> 
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/orgs/espressopp/teams/discussion/discussions/14>, or 
> unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AADZSLGPIDGZ5B56GDFOWHTWGI2EHANCNFSM6AAAAAARU4KIGQ>.
> You are receiving this because you are subscribed to this thread.Message 
> ID: ***@***.***>
> 

-- 
Max-Planck-Institut für Polymerforschung
Dr. Torsten Stuehn (Head of Scientific IT)
Ackermannweg 10
55128 Mainz
Tel +49-(0)6131-379268

**Comment by xzhh** [comment]
So lets move to 16:30 for tomorrow.

**Comment by mrampp** [comment]
Same for me.

schoene Gruesse / best regards,
Markus Rampp

--
Dr. Markus Rampp
Max Planck Computing and Data Facility
Garching, Germany
Phone: +49/89/3299-1324

**Comment by mrampp** [comment]
16:30 (CEST) *today* is fine with me

Markus

On Thu, Apr 20 2023, Torsten Stuehn wrote:
> Dear all,
> 
> due to the a CECAM (online-) talk that I would like to attend, I will 
> only be available after 16:30 and though would like to shift the meeting 
> until that time.
> 
> Would that be possible ? (if not, I would simply join the meeting at 16:30)
> 
> Best regards,
> 
> Torsten
> 
> 
> On 11/2/22 10:47, Zhen-Hao Xu wrote:
> > Dear all
> > 
> > The starting date of winter time may differ in different countries. If 
> > there's no other suggestions, lets stick to 4.00 pm (CET) for the weekly 
> > meeting and pay attentions to your local time.
> > 
> > Zhenhao
> > 
> > —
> > Reply to this email directly, view it on GitHub 
> > <https://github.com/orgs/espressopp/teams/discussion/discussions/14>, or 
> > unsubscribe 
> > <https://github.com/notifications/unsubscribe-auth/AADZSLGPIDGZ5B56GDFOWHTWGI2EHANCNFSM6AAAAAARU4KIGQ>.
> > You are receiving this because you are subscribed to this thread.Message 
> > ID: ***@***.***>
> > 
> 
> -- 
> Max-Planck-Institut für Polymerforschung
> Dr. Torsten Stuehn (Head of Scientific IT)
> Ackermannweg 10
> 55128 Mainz
> Tel +49-(0)6131-379268
> 
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/espressopp/espressopp/discussions/468#discussioncomment-5673959
> You are receiving this because you commented.
> 
> Message ID: ***@***.***>

-- 
best regards / schoene Gruesse,
Markus Rampp

--  
---------------------------------------------------------
Dr. Markus Rampp
Max Planck Computing and Data Facility (MPCDF)
Giessenbachstr. 2, D-85748 Garching bei Muenchen, Germany
Phone: +49-89-3299-1324           FAX: +49-89-3299-1301
E-Mail: ***@***.*** WWW: www.mpcdf.mpg.de
---------------------------------------------------------
